### DISCUSSION TITLE
Dealing with GOES projection coordinates

### DISCUSSION THREAD

**Discussion by blaylockbk** [blaylockbk/goes2go#30] [General]
I'd like to start a thread related to dealing with GOES projection coordinates. Several people have asked me recently about this, and I'd like to bring all those people together here to talk about this topic.

* How to parse the projection information to Latitude/Longitude (for each domain type, FullDisk, CONUS, Mesosector)
* Links to resources with helpful information
* Plotting data with Cartopy
* etc.

**Comment by blaylockbk** [comment]
## Unidata Tutorial
https://unidata.github.io/python-gallery/examples/mapping_GOES16_TrueColor.html

**Comment by bryanguarente** [comment]
Some of the GOES L2 products on AWS don't have all the necessary information you are using in your FOV.  I am seeing at least three products that the FOV doesn't work for: DMWs, Reflected Shortwave Radiation Top of Atmosphere, and Downward Shortwave Radiation.  These seem to break the FOV accessor because they have a G['goes_lat_lon_projection'] instead of a G['goes_imager_projection'] but have some of the other FOV access points you are expecting.

**Comment by blaylockbk** [comment]
Found this
https://github.com/uba/goes-latlon

**Reply by blaylockbk** [reply]
This is an old notebook that might be outdated, but it might point you in the right direction. 

https://github.com/blaylockbk/goes2go/blob/main/latlon_grids/create_standard_GOES_latlon_grids.ipynb
